In-house patch to remove the dependency on unittest2. Python 2.7
includes unittest, which is equivalent to unittest2 for Python 2.6. If
this line is present, stevedore will fail to find the pip package for
it, which causes a failure.

We also remove python-mimeparse, an optional dependency that didn't have
a release for many years and is not needed for testing in Solaris.

--- testtools-2.4.0/requirements.txt
+++ testtools-2.4.0/requirements.txt
@@ -1,9 +1,4 @@
 pbr>=0.11
 extras>=1.0.0
 fixtures>=1.3.0
-# 'mimeparse' has not been uploaded by the maintainer with Python3 compat
-# but someone kindly uploaded a fixed version as 'python-mimeparse'.
-python-mimeparse
-unittest2>=1.0.0
-traceback2
 six>=1.4.0
--- testtools-2.4.0/testtools.egg-info/requires.txt
+++ testtools-2.4.0/testtools.egg-info/requires.txt
@@ -1,12 +1,4 @@
 extras>=1.0.0
 fixtures>=1.3.0
 pbr>=0.11
-python-mimeparse
 six>=1.4.0
-traceback2
-unittest2>=1.0.0
-
-[test]
-testresources
-testscenarios
-unittest2>=1.1.0
